mod/config/config.go

```diff
@@ -53,15 +55,17 @@ type Config struct {
 	Engine engineclient.Config `mapstructure:"engine"`
 	// KZG is the configuration for the KZG blob verifier.
 	KZG kzg.Config `mapstructure:"kzg"`
+	// Logger is the configuration for the logger.
+	Logger logger.Config `mapstructure:"logger"`
```

Comment:
tbh I think this is overkill, should be configarable via the node-builder, but having it in a config file is too much

Reply:
yeah true i agree, was originally thinking some basic options such as time format could be configurable through cli